addons/web/static/tests/helpers/mock_server.js

```diff
@@ -326,12 +328,7 @@ export class MockServer {
                 relModel = field.relation;
                 relatedModels.add(relModel);
                 // inline subviews: in forms if field is visible and has no widget (1st level only)
-                if (
-                    inFormView &&
-                    level === 0 &&
-                    !node.getAttribute("widget") &&
-                    !node.getAttribute("invisible")
-                ) {
+                if (inFormView && level === 0 && !node.getAttribute("widget") && !isInvisible) {
```

Comment:
if invisible is not found as an attr but in modifiers instead, it is not taken into consideration. Is it normal?

Reply:
The `modifiers` attribute is generated by the framework, no one is supposed to define it manually. Was it your question?